third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/container/resource_container_cluster.go

```diff
@@ -2680,6 +2687,9 @@ func resourceContainerClusterRead(d *schema.ResourceData, meta interface{}) erro
 	if err := d.Set("enable_multi_networking", cluster.NetworkConfig.EnableMultiNetworking); err != nil {
 		return fmt.Errorf("Error setting enable_multi_networking: %s", err)
```

Comment:
AGENT: The error message here is a bit generic. Please consider wrapping the error with more context, such as the cluster name, to make debugging easier if this `d.Set` call fails. For example: `fmt.Errorf("Error setting enable_cilium_clusterwide_network_policy for cluster %s: %w", d.Id(), err)`.